qiskit/circuit/library/standard_gates/z.py

```diff
@@ -215,7 +223,15 @@ def _define(self):
         self.definition = qc
 
     def inverse(self, annotated: bool = False):
-        """Return inverted CZ gate (itself)."""
+        """Return inverted CZ gate (itself).
+
+        Args:
+            annotated: indicates whether the inverse gate can be implemented
+                as an annotated gate.
```

Comment:
```suggestion
            annotated: Typically when set to ``True`` this is used to return an :class:`.AnnotatedOperation` 
                with an inverse modifier set instead of a concrete :class:`.Gate`. However, for this class this
                argument is ignored as this gate is self inverse.
```